server/app/crud.py

```diff
@@ -168,12 +180,14 @@ def get_vectors_for_user(database: Session, user: schemas.User) -> List[model.Ve
     return database.query(model.Vector).filter(model.Vector.users.any(id=user.id)).all()
 
 
-def get_all_vectors(database: Session) -> List[model.Vector]:
+def get_all_vectors(
+    database: Session, offset: int = 0, limit: int = 0
+) -> List[model.Vector]:
     """
     Returns every vector in the Vectors table (necessary for adding child-parent relations)
     Should only be used for importing the Level 1 elements from genbank files and csv!
     """
```

Comment:
I don't get why `get_all_vectors` should only be used for importing the level 1 elements from genbank files and csv.
Once they're in the DB, can you a distinction between those that were imported from genbank and those that were generated by GG2?